### PR TITLE
[1LP][RFR] Fixed vm provisioning error in quota test

### PR DESCRIPTION
**Pull request by ganeshhubale**
{{ pytest: cfme/tests/cloud/test_quota_tagging.py::test_quota_tagging_cloud_via_lifecycle --use-provider complete }}

### CODE REVIEW

File: cfme/common/vm.py [hunk 1]
```diff
@@ -531,6 +531,7 @@ def create(self, vm_name, provider, form_values=None, cancel=False, check_existi
                              handle_exception=True, num_sec=600)
                 elif override:
                     logger.info('Overriding exception to check failure condition.')
+                    return vm
```
Comment (digitronik) [inline cfme/common/vm.py]: Question: For which provider you are getting `Exception` error? I am suspect about returning `vm` object in `override`.

Comment (ganeshhubale) [reply cfme/common/vm.py]: I am getting Exception error for **openstack** provider.

Comment (digitronik) [reply cfme/common/vm.py]: Required: 
Then no need to return `vm` object `override` will handle an exception. As per your test no need of `vm object`. debug your failures.

**Comment by ganeshhubale** [comment]
@digitronik 
PR is updated with required changes please take a look at it.
Thank you.

**Pull request merged**